uninstall/uninstall_linux.go

```diff
@@ -0,0 +1,90 @@
+package uninstall
+
+import (
+	"time"
+	"io/ioutil"
+	"os/exec"
+	"github.com/pufferpanel/pufferd/logging"
+	"syscall"
+)
+
+
+func StartProcess(){
+	killDaemon()
+	deleteFiles()
+	deleteUser()
+}
+
+func killDaemon(){
+	exec.Command("killall", "-15" "-u", "pufferd").Run()

```

Comment:
I would rather use systemctl directly so it is clear we are talking about killing it through that, the kill -15 is not clear that it's doing that.